odl/util/utility.py

```diff
@@ -157,11 +156,12 @@ def dedent(string, indent_str='   ', max_levels=None):
 
     lines = string.splitlines()
 
-    # Determine common (minumum) number of indentation levels, capped at
+    # Determine common (minimum) number of indentation levels, capped at
     # `max_levels` if given
     def num_indents(line):
         max_num = int(np.ceil(len(line) / len(indent_str)))
 
+        i = 0
```

Comment:
Otherwise undefined if loop does not run (`max_num == 0`)

Reply:
Add a comment above to clarify this?